XenAdmin/Dialogs/AcceptEuaDialog.resx

```diff
@@ -0,0 +1,959 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
about

> The parent should be the wizard, not the main window, otherwise it may appear hidden behind the wizard.

I don't see a mechanism to do this easily (unless I just pass the `Control` all the way from the precheck page to the `Problem`. Other checks seem to just use the mainwindow. Maybe such a mechanism could be added as part of a possible refactoring of the `Check`s?

If you want to I can pass the wizard's form all the way, but it won't look very nice :/

Reply:
> Other checks seem to just use the mainwindow
It depends on the case - and some cases may not even be correct (another check audit 😞 ).

In this case it is needed because if you open the EUA dialog, then click on different application on the same screen and then back to XC, the EUA dialog is hidden behind the wizard and because it is modal you can't click on the wizard, you need to click on the main window behind it to see the EUA dialog again, which is not great UX. Yes, we need to pass in the control (see for example the DMC check - when I added that I had seen an issue with adding it to the base class, I can't remember right now what exactly it was, but we can look at it again, maybe in a different PR).